whylog/assistant/__init__.py

```diff
@@ -21,21 +21,19 @@ class RegexAssistant(AbstractAssistant):
     One RegexAssistant per one entering Rule.
     """
 
-    def __init__(self, effect, line_objects=tuple()):
+    def __init__(self):
+        self.lines = {}
```

Comment:
both lines and regexes are indexed by the same identifier. They are added and deleted simultaneously. It indicates that probably `RegexAssistant` should have a collection of a class which wraps both (the line and the regex).